book/website/reproducible-research/vcs.md

```diff
@@ -13,49 +13,76 @@
+the data was used to produce a particular figure.
+This provenance information is enabled and facilitated when both the data, the code aand the figure files are under versioning.
 
-Version control helps us understand what changes we made in the past or why we did a specific analysis in the way we did it, even weeks or months later.
-With the help of comments and commit messages, each version can explain what changes it contains compared to the previous versions.
+In addition, version control creates **version history** to help us understand 
+what changes were made, or why a specific analysis was run, even weeks or months later.
+With the help of comments and commit messages in git, for instance, each version can explain what changes it contains compared to the previous versions.
 This is helpful when we share our analysis (not only data), and make it auditable or **reproducible** - which is good scientific practice.
 
-A version control system neatly stores a history of changes and who made them, so while it is still easy to access them, your working directory is not cluttered by the debris of previous versions that are necessary to keep just in case.
-Similarly, with version control, there is no need to leave chunks of code commented should you ever need to come back to an old version again.
+A version control system **neatly hide older versions** of the data. 
+So your working directory is not cluttered by the debris of previous versions, while they remain accessible, in case you need them
+Similarly, with version control, there is no need to leave unused chunks of code  should you ever need to come back to an old version again.
 
 
-Finally, version control is invaluable for collaborative projects where different people work on the same code simultaneously and build on each other's work.
-It allows the changes made by different people to be tracked and can automatically combine people's work while saving a great deal of painstaking effort to do so manually.
-Using version control for your research project means that your work is totally transparent, and because all your actions are recorded, it enables others to reproduce your studies.
-Moreover, version control hosting services such as {ref}`GitHub<cl-github-novice-motivation>` provide a way to communicate and collaborate in a more structured way, such as in pull requests, code reviews, and issues.
+Finally, version control is invaluable for collaborative projects where different people work on the same data or code simultaneously and build on each other's work.
+Using a version contols system, **Changes made by different people can be tracked and often automatically combined**, saving a great deal of painstaking efforts.
+Using version control for your research project means that your work is more transparent. Because all your actions are recorded, your studies are easier to reproduce and build upon.
```

Comment:
```suggestion
Using version control for your research project means that your work is more transparent. 
Because all your actions are recorded, your studies are easier to reproduce and build upon.
```